vulcanai2/tests/models/test_metrics.py

```diff
@@ -2,13 +2,42 @@
 import numpy as np
 import torch
 from vulcanai2.models.metrics import Metrics
+from vulcanai2.models.cnn import ConvNet
+from vulcanai2.models.dnn import DenseNet
+from vulcanai2.models.ensemble import SnapshotNet
+from torch.utils.data import TensorDataset, DataLoader
 
 class TestMetrics:
     @pytest.fixture
     def metrics(self):
         return Metrics(
             num_class=2
```

Comment:
you have num_classes as 2 here and 6 when you make the network? we might as well remove it from metrics no?